modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -341,14 +463,12 @@ using GConstGIEModel = ade::ConstTypedGraph
     , IECallable
     >;
 
-using Views = std::vector<std::unique_ptr<cv::MediaFrame::View>>;
-
-inline IE::Blob::Ptr extractBlob(IECallContext& ctx, std::size_t i, Views& views) {
-    switch (ctx.in_shapes[i]) {
+inline IE::Blob::Ptr extractBlob(IECallContext& ctx, std::size_t i) {
+    switch (ctx.inShape(i)) {
```

Comment:
@TolyaTalamanov 
There are several warnings in Windows builds with IE.
Please take a look.

```
C:\build\precommit_custom_windows\opencv\modules\gapi\src\backends\ie\giebackend.cpp(467): warning C4267: 'argument': conversion from 'size_t' to 'int', possible loss of data [C:\build\precommit_custom_windows\build\modules\gapi\opencv_gapi.vcxproj]
C:\build\precommit_custom_windows\opencv\modules\gapi\src\backends\ie\giebackend.cpp(631): warning C4267: 'argument': conversion from 'size_t' to 'int', possible loss of data [C:\build\precommit_custom_windows\build\modules\gapi\opencv_gapi.vcxproj]
C:\build\precommit_custom_windows\opencv\modules\gapi\src\backends\ie\giebackend.cpp(660): warning C4267: 'argument': conversion from 'size_t' to 'int', possible loss of data [C:\build\precommit_custom_windows\build\modules\gapi\opencv_gapi.vcxproj]
```

Reply:
Got it! Will be fixed in there: https://github.com/opencv/opencv/pull/19533